### PR TITLE
Add backingstore to disk

### PR DESCRIPTION
**Pull request by chloerh**

### CODE REVIEW

**Comment by kylazhang** [comment]
How about using BackingStore as TypedDeviceBase and Source as UntypedDeviceBase to avoid the alert

**Comment by dzhengfy** [comment]
@chloerh I also suggest to add some background information for 'backingstore' in the PR description field.

**Comment by chloerh** [comment]
> How about using BackingStore as TypedDeviceBase and Source as UntypedDeviceBase to avoid the alert

But other nestedxml classes such as 'Reservations' inherit from base.base.LibvirtXMLBase, I think we'd better keep them consistent

File: virttest/libvirt_xml/devices/disk.py [hunk 1]
```diff
@@ -409,3 +414,86 @@ def __init__(self, virsh_instance=base.base.virsh, reservations_managed=""):
                                    tag_name='source', attribute='mode')
             super(self.__class__, self).__init__(virsh_instance=virsh_instance)
             self.xml = '<reservations/>'
+
+    class BackingStore(base.base.LibvirtXMLBase):
+        """
+        BakingStore of disk device XML class
+
+        type:
+            string
```
[the remark this answers is not in the document]
Comment (chloerh) [reply virttest/libvirt_xml/devices/disk.py]: Updated

File: virttest/libvirt_xml/devices/disk.py [hunk 2]
```diff
@@ -409,3 +414,86 @@ def __init__(self, virsh_instance=base.base.virsh, reservations_managed=""):
...
+
+            dev:
+                string
+            protocal:
+                string
+            name:
+                string
+            host:
+                dict
+            """
```
[the remark this answers is not in the document]
Comment (chloerh) [reply virttest/libvirt_xml/devices/disk.py]: Updated

**Comment by chloerh** [comment]
> @chloerh I also suggest to add some background information for 'backingstore' in the PR description field.

I already has the similar description as other xml classes.

**Pull request merged**